--- /usr/sbin/multistrap	2014-02-21 23:27:42.000000000 -0800
+++ multistrap	2017-02-16 10:22:26.675399614 -0800
@@ -986,7 +986,7 @@
 	}
 	# reinstall set
 	foreach my $reinst (sort @reinstall) {
-		system ("$str $env chroot $dir apt-get --reinstall -y $forceyes install $reinst");
+		system ("$str $env chroot $dir apt-get --reinstall -y install $reinst");
 	}
 	&run_native_hooks_end(sort @{$hooks{'N'}}) if (defined $hooks{'N'});
 	return $retval;
